src/components/form/Input/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { InputHTMLAttributes } from 'react';
+import classNames from 'classnames';
+
+import classes from './input.module.css';
+
+export interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
```

Comment:
What do you think about allowing a validator here? For example, salesforce subdomains are supposed to be all lowercase, between 4-19 characters, etc ([ref](https://help.salesforce.com/s/articleView?id=sf.mth_set_up_your_mth_subdomain_and_namespace.htm&type=5)), and we could save a roundtrip for the user if we allow pre-API validation. The optionality to have a validator do that would be nice

Reply:
Yeah inline-validation and advanced form state (isDirty, isTouched etc) is out of scope right now given the size of the refactor/rewrite. Definite consideration in the future tho.